doc/UsersGuide/source/CodeOverview.rst

```diff
@@ -45,12 +45,12 @@ The umbrella repository for the UFS Weather Model is named ufs-weather-model.  U
 
    ufs-weather-model/
    ├── FMS                                     https://github.com/NOAA-GFDL/FMS
-   ├── FV3                                     https://github.com/NCAR/fv3atm
-   │   ├── atmos_cubed_sphere                  https://github.com/NCAR/GFDL_atmos_cubed_sphere
+   ├── FV3                                     https://github.com/NOAA-EMC/fv3atm
+   │   ├── atmos_cubed_sphere                  https://github.com/NOAA-EMC/GFDL_atmos_cubed_sphere
```

Comment:
GFDL_atmos_cubed_shere in public release is pointing to NOAA-GFDL repo.

Reply:
Tks This has been addressed.

Reply:
I see line 36 is changed, but line 49 is still listing NOAA-EMC, not NOAA-GFDL.